modules/pubmaticBidAdapter.js

```diff
@@ -861,8 +861,8 @@ export const spec = {
             utils.logError(`${LOG_WARN_PREFIX}: no context specified in bid. Rejecting bid: `, bid);
             return false;
           }
-          if (bid.mediaTypes[VIDEO].context === 'outstream' && !utils.isStr(bid.params.outstreamAU)) {
-            utils.logError(`${LOG_WARN_PREFIX}: for "outstream" bids outstreamAU is required. Rejecting bid: `, bid);
+          if (bid.mediaTypes[VIDEO].context === 'outstream' && !utils.isStr(bid.params.outstreamAU) && !bid.hasOwnProperty('renderer')) {
```

Comment:
This looks good to me. You may want to consider eventually looking within bid.mediaTypes[VIDEO] for the renderer as Publishers do have the ability to define on this level for the specific mediaType context in multiformat adunits: https://docs.prebid.org/dev-docs/show-outstream-video-ads.html#renderers